jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -18,14 +18,14 @@
 
 package org.eclipse.jetty.http;
 
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.CASE_INSENSITIVE_FIELD_VALUE_CACHE;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.FIELD_COLON;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.FIELD_NAME_CASE_INSENSITIVE;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.METHOD_CASE_SENSITIVE;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.NO_FIELD_FOLDING;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.NO_HTTP_0_9;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.NO_WS_AFTER_FIELD_NAME;
-import static org.eclipse.jetty.http.HttpParser.ViolationSection.TRANSFER_ENCODING_WITH_CONTENT_LENGTH;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.CASE_INSENSITIVE_FIELD_VALUE_CACHE;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.FIELD_COLON;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.FIELD_NAME_CASE_INSENSITIVE;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.METHOD_CASE_SENSITIVE;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.NO_FIELD_FOLDING;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.NO_HTTP_0_9;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.NO_WS_AFTER_FIELD_NAME;
+import static org.eclipse.jetty.http.HttpParser.RFC7230SpecReference.TRANSFER_ENCODING_WITH_CONTENT_LENGTH;
```

Comment:
I know it is "standard" but I really don't like having the static imports as the first ones.   Not really part of this review, but I think we should swap to having static imports as the last imports listed.